examples/language-modeling/run_clm.py

```diff
@@ -592,9 +592,15 @@ def group_texts(examples):
             )
 
     if training_args.do_train:
+
+        def tensor_mapper(x):
+            return {i: torch.tensor(x[i], dtype=torch.int32) for i in x}
+
         if "train" not in tokenized_datasets:
             raise ValueError("--do_train requires a train dataset")
         train_dataset = lm_datasets["train"]
+        if training_args.resume_from_checkpoint != '':
```

Comment:
`training_args.resume_from_checkpoint` may be None, as it's the default value if it is not specified. So I think we should do:
```suggestion
        if training_args.resume_from_checkpoint is not None and training_args.resume_from_checkpoint != '':
```